### DISCUSSION TITLE
Do I need to use the same email account for AWS and Seller Central?

### DISCUSSION THREAD

**Discussion by johntiror** [amzn/selling-partner-api-models#2812] [General]
Hello,

I created an app on Seller Central using EMAIL_1 and I created an AWS account using a different EMAIL_2. I followed every step, created the IAM role, priviledge, user according to the given documentations. Now everytime i receive following error message (403):

```
{
    "errors": [
        {
            "message": "Access to requested resource is denied.",
            "code": "Unauthorized",
            "details": ""
        }
    ]
}
```

Is it because, the AWS and SC Accounts are different accounts with different email addresses or is there another possible reason?
The application is in draft.

Thanks

**Comment by sendurangr** [comment]
Hi am also getting the same issue, I'm also using two different accounts, have you found the root cause for this?
